SocialFilter/sections/specific.txt

```diff
@@ -89,6 +89,11 @@ gizbot.com##.gibot-prod-specs-share
 gizbot.com##.footer-article-social-share
 gizbot.com##.one-head-share-block
 gizbot.com##.pssocialbtn
+informazione.it##a[href^="https://www.twitter.com/"][target="_blank"]
+informazione.it##a[href^="https://www.facebook.com/"][target="_blank"]
+informazione.it##.soc-btns
+informazione.it###tfr-btns
+informazione.it###btnStrip
```

Comment:
we do not block links to blogs
Also these elements not reproduced
```
informazione.it##a[href^="https://www.twitter.com/"][target="_blank"]
informazione.it##a[href^="https://www.facebook.com/"][target="_blank"]
```

Reply:
These were also links to fb and tw blogs, removed them